### PR TITLE
docs: testing plan for exam configuration

### PR DESCRIPTION
**Pull request by Zacharis278**
**JIRA:** [MST-1657](https://2u-internal.atlassian.net/browse/MST-1657)

**Description:** Document testing steps for exam setup and provider configuration. (Dev milestone 2)

The plan is to have a separate file for each milestone/feature set to keep things organized. That is, assuming we choose to continue this approach in future milestones.

### CODE REVIEW

File: docs/test_plan/exam_configuration.rst [hunk 1]
```diff
@@ -0,0 +1,189 @@
+Exam Setup and Configuration
+============================
+
+Exam Configuration (edx-exams)
+------------------------------
+
+Prerequisite Steps:
+   #. Make sure `course_apps.exams_ida` CourseWaffleFlag is enabled for your test course
+   #. An LTI proctoring provider should be chosen for the course
```
Comment (ashultz0) [inline docs/test_plan/exam_configuration.rst:9]: we're presumably using a fake one? can we put setup instructions in a section here, or maybe in another doc?

Comment (Zacharis278) [reply docs/test_plan/exam_configuration.rst:9]: humm yeah maybe I can lean on the read the docs pages more here and elsewhere in this doc

Comment (schenedx) [reply docs/test_plan/exam_configuration.rst:9]: Agree. As this is written, I won't know how to do this

**Review by schenedx — APPROVED** [review]
👍  Regardless of my comment, this document is very useful for anyone who reads and use the code.

**Pull request merged**